lasagne/layers/base.py

```diff
@@ -92,7 +92,7 @@ def get_params(self, **tags):
             result = [param for param in result
                       if not (self.params[param] & exclude)]
 
-        return result
+        return utils.collect_shared_vars(result)
```

Comment:
(Although we could also change line 81 above to not listify `result`, for unmeasurably improved performance. Maybe I'll still do that.)